pack8s/internal/pkg/podman/podman_test.go

```diff
@@ -304,6 +304,7 @@ var _ = Describe("podman", func() {
 			Expect(err).To(BeNil())
 
 			images, err := handler.ListImages()
+			Expect(err).To(BeNil())
```

Comment:
Might be better to use `Expect(err).ShouldNot(HaveOccurred())`
but i understand that it will pop out because the rest use `Expect(err).To(BeNil())`

Reply:
Thanks for your review and sorry for my late reply, I have updated the error assert. PTAL.

Reply:
Thanks, I meant that I was wondering myself which style we should use,
since all the file has already `Expect(err).To(BeNil())`
I am not sure we want one place to pop up differently, sorry for the misunderstanding

Reply:
Thanks, PTAL.